python/paddle/fluid/framework.py

```diff
@@ -2055,8 +2056,9 @@ def _copy_dist_param_info_from(self, other):
                             "Program")
         self._is_distributed = other._is_distributed
         self._is_chief = other._is_chief
-        self._slice_vars_and_attrs = other._slice_vars_and_attrs
+        self._slice_vars_overview = other._slice_vars_overview
```

Comment:
what does `overview` mean?

Reply:
The distribution of all parameters is recorded in the parameter ```_slice_vars_overview ```.